index.js

```diff
@@ -66,8 +68,9 @@ function Converter (sm, opts) {
 
   if (opts.isFileComment) sm = readFromFileMap(sm, opts.commentFileDir);
   if (opts.hasComment) sm = stripComment(sm);
-  if (opts.isEncoded) sm = decodeBase64(sm);
-  if (opts.isJSON || opts.isEncoded) sm = JSON.parse(sm);
+  if (opts.encoding === 'base64') sm = decodeBase64(sm);
```

Comment:
Let's add some braces here .. this is getting hard to parse

Reply:
Done!